analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentService.java

```diff
@@ -30,4 +30,46 @@ GAV getProject() {
 
 		String getAlignedVersionOfGav(GAV gav);
 	}
+
+	interface RequestCustomizer {
+
+		Request customize(Request request);
+
+		// Integer.MIN_VALUE is the max order
```

Comment:
Could you clarify the comment to specify how the order works with respect to priority of customization, please?

Reply:
Sure thing

Reply:
Done